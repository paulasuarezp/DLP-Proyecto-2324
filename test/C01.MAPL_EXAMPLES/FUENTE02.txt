// Operaciones con reales

float f;

read f;

print 1 + f * 2;
