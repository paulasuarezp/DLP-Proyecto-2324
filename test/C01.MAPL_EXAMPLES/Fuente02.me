//*
Ejemplo Fuente02.
Operaciones con reales
*//

class Fuente02;
	global
		vars
			f : DOUBLE;
		
	create
		main; 
			
	feature	main is
		/* local
			
		do
		read f;

		print 1.0 + f * 2.0;
	end /*main
	
end /* Fuente02

run main();