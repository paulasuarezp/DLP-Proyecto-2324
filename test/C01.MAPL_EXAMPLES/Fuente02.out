' Compilador mini Eiffel.
' Aquilino A. Juan.
' Versi√≥n 2024.2 (c)
#source "Fuente02.me"
#global f: float
#func main
#ret void
CALL main
HALT

#line 14
main:

#line 18
PUSHA 0
INF
STOREF

#line 20
PUSHF 1.0
PUSHA 0
LOADF
PUSHF 2.0
MULF
ADDF
OUTF

#line 20
RET 0, 0, 0
